### PR TITLE
Calendar

### PR DESCRIPTION
**Pull request by wJoenn**
It's not completed yet because it needs to be integrated with Rails to take the existing sessions into consideration to design them differently than the days without sessions but that's outside the scope of this issue so it'll be done in another one later
image

### CODE REVIEW

**Review by k-kenney — COMMENTED** [review]
The calendar looks really good. I'm not quite sure what to do with the red vite_javascript part and don't want to mess anything up.  
Screen Shot 2023-05-08 at 8 43 40 AM

File: app/frontend/entrypoints/application.js [hunk 1]
```diff
@@ -1,24 +1,9 @@
...
-//
-// If you want to use .jsx or .tsx, add the extension:
-//     <%= vite_javascript_tag 'application.jsx' %>
-
-console.log('Visit the guide for more information: ', 'https://vite-ruby.netlify.app/guide/rails')
-
 // Example: Load Rails libraries in Vite.
 //
-// import * as Turbo from '@hotwired/turbo'
-// Turbo.start()
```
Comment (k-kenney) [inline app/frontend/entrypoints/application.js:21]: Just not sure what to do about this part. The Calendar design looks nice so far though.

**Comment by wJoenn** [comment]
This application.js is automatically created by Vite when it's installed.
It adds a couple console.log to indicate the javascript is correctly linked when trying the server and it also leaves a bunch of instructions on how to initialize other stuffs.

We don't need those comments not console logs so I just deleted them.
It's not code that would do stuff anyway

File: app/views/pages/home.html.erb [hunk 2]
```diff
@@ -1,3 +1,10 @@
-<div class="container">
-  <%= render "pages/partials/membership" %>
```
Comment (k-kenney) [inline app/views/pages/home.html.erb:2]: It just looks like there's a repeat here. One should be deleted, right?

**Review by k-kenney — APPROVED** [review]

File: app/views/pages/home.html.erb (hunk 2 above)
Comment (wJoenn) [reply app/views/pages/home.html.erb:2]: The red lines are what is deleted and the green lines are what's been added
So this looks like this in reality :
```
<div id="home">
  <div class="container">
    <%= render "pages/partials/membership" %>

    <section id="planning">
      <h2>Planning</h2>
      <%= render "pages/partials/calendar" %>
    </section>
  </div>
</div>
```

File: app/frontend/entrypoints/application.js (hunk 1 above)
Comment (wJoenn) [reply app/frontend/entrypoints/application.js:21]: This application.js is automatically created by Vite when it's installed.
It adds a couple console.log to indicate the javascript is correctly linked when trying the server and it also leaves a bunch of instructions on how to initialize other stuffs.

We don't need those comments not console logs so I just deleted them.
It's not code that would do stuff anyway

Comment (wJoenn) [reply app/frontend/entrypoints/application.js:21]: The red lines just mean I deleted all of that stuff

**Pull request merged**